pkg/cloud/aws/scanner/scanner.go

```diff
@@ -160,11 +160,11 @@ func createState(freshState *state.State, awsCache *cache.Cache) (*state.State,
 	return fullState, nil
 }
 
-type defsecLogger struct {
+type DebugLogger struct {
 }
 
-func (d *defsecLogger) Write(p []byte) (n int, err error) {
-	log.Logger.Debug("[defsec] " + strings.TrimSpace(string(p)))
+func (d *DebugLogger) Write(p []byte) (n int, err error) {
+	log.Logger.Debug("[aws] " + strings.TrimSpace(string(p)))
```

Comment:
How about passing a [named logger](https://pkg.go.dev/go.uber.org/zap#Logger.Named) as a field?   Zap will take care of adding the path to the logger name itself